library.properties

```
@@ -3,7 +3,7 @@ version=1.0.9
 author=Lenard George, Pablo Marquínez
 maintainer=Arduino <info@arduino.cc>
 sentence=Arduino Sensor Kit
-paragraph=This library wraps all the libraries needed to use the Sensor kit breakout board: Oled display, Humidity and Temperature sensor, Pressure sensorand Accelerometer .
+paragraph=This library wraps all the libraries needed to use the Sensor kit breakout board: Oled display, Humidity and Temperature sensor, Pressure sensor and Accelerometer.

```

```suggestion
paragraph=This library wraps all the libraries needed to use the Sensor Kit breakout board: OLED display, humidity and temperature sensor, pressure sensor and accelerometer.
```

While you're at it, might as well also correct the capitalization and case errors.

References:

- https://store.arduino.cc/products/arduino-sensor-kit-base
- https://en.wikipedia.org/wiki/OLED

---

Good point!